packages/app/src/runs/RunsEmpty.vue

```diff
@@ -58,7 +59,9 @@ const firstRecordKey = computed(() => {
     : '<record-key>'
 })
 const recordCommand = computed(() => {
-  return `cypress run --record --key ${firstRecordKey.value}`
+  const componentFlagOrSpace = props.gql.currentTestingType === 'component' ? ' --component ' : ' '
```

Comment:
You could do a `['arg1', 'arg2'].join(' ')` here if you wanted to to remove the space addition.